katdal/test/test_chunkstore_s3.py

```diff
@@ -120,7 +120,14 @@ def array_name(self, path):
         bucket = 'katdal-unittest'
         return self.store.join(bucket, path)
 
+    @timed(0.1 + 0.1 + 1 + 0.05)
```

Comment:
I would rather split this into two separate timed tests. At the moment if it fails you won't know which one was slow.

Reply:
Point taken.